src/app/shared/dtos/vehicle-dto.ts

```diff
@@ -0,0 +1,26 @@
+import { Transmission } from "@app/shared/models/transmission";
+import { Vehicle } from "@app/shared/models/vehicle";
+
+export class VehicleDto {
+    constructor(
+        public aggregateId: number,
+        public manufacturer: string,
+        public model: string,
+        public capacity: number,
+        public power: number,
+        public transmission: Transmission,
+        public fleetId: number,
+        public productionYear: number
+    ) { }
+
+
+    public static toVehicleDto(entity: Vehicle) {
```

Comment:
jak już to `from` -> VehicleDto.from(vehicle);

Reply:
zmieniono.